app/services/articles/feed.rb

```diff
@@ -176,7 +205,7 @@ def score_comments(article)
 
     def globally_hot_articles(user_signed_in)
       hot_stories = published_articles_by_tag.
-        where("score > ? OR featured = ?", 8, true).
+        where("score > ? OR featured = ?", 7, true).
```

Comment:
This is just switching this magic number down by one, one more tweak not 100% related to the PR, but I think this will be good.